### PR TITLE
Full material color palette

### PR DESCRIPTION
**Pull request by alorma**
[Review on Reviewable](https://reviewable.io/reviews/mikepenz/materialize/2)

### CODE REVIEW

File: library/src/main/java/com/mikepenz/materialize/color/Material.java [hunk 1]
```diff
@@ -48,5 +48,759 @@ public int getAsResource() {
...
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
+    public enum Monochrome implements IColor {
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/materialize/color/Material.java:777]: Not yet sure if Monochrome is the best thing. I would split up black and white as all other colors.

File: library/src/main/java/com/mikepenz/materialize/color/Material.java [hunk 2]
```diff
@@ -48,5 +48,759 @@ public int getAsResource() {
...
+            return Color.parseColor(color);
+        }
+
+        @Override
+        public int getAsResource() {
+            return resource;
+        }
+    }
+
 }
```
Comment (mikepenz) [inline library/src/main/java/com/mikepenz/materialize/color/Material.java:806]: We also need a special enum for specific colors like those defined here.

http://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0Bx4BSt6jniD7eFBPX3BGRW04SE0/style_color_uiapplication_alpha1.png

http://material-design.storage.googleapis.com/publish/material_v_4/material_ext_publish/0Bx4BSt6jniD7dTJwdmYtcnRrMUE/style_color_uiapplication_alpha2.png

Those need to be ARGB values

Comment (alorma) [inline library/src/main/java/com/mikepenz/materialize/color/Material.java:806]: Not sure about how can we apply it, i will check in a future PR

Comment (mikepenz) [inline library/src/main/java/com/mikepenz/materialize/color/Material.java:806]: Here for white: 
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/res/values/colors.xml#L51

And here for black:
https://github.com/mikepenz/MaterialDrawer/blob/develop/library/src/main/res/values/colors.xml#L12

**Pull request merged**